python/ray/workflow/dag_to_workflow.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any
+
+from ray import workflow
+
+from ray.experimental.dag import DAGNode, FunctionNode
+from ray.experimental.dag.input_node import InputAtrributeNode, DAGInputData
+
+
+def _process_input_value(v: Any, input_context):
+    if isinstance(v, FunctionNode):
+        return _transform_dag_recursive(v, input_context)
```

Comment:
Did we do deduplicate here? For example, if the same inputs are given, will we end up with two workflow steps?

Reply:
good catch!